src/tsickle_compiler_host.ts

```diff
@@ -209,6 +214,15 @@ export class TsickleCompilerHost implements ts.CompilerHost {
     return tscSourceMapGenerator.toString();
   }
 
+  combineInlineSourceMaps(filePath: string, tscOutputText: string): string {
```

Comment:
Can you add some jsdoc to this about the parameters?  (e.g. what is tscOutputText)